### PR TITLE
24/cluster click

### PR DESCRIPTION
**Pull request by hagoptaminian**
merging blocked by PR #23

### CODE REVIEW

**Comment by juancarlosfarah** [comment]
@hagoptaminian, you can now rebase this and mark it ready for review.

**Comment by hagoptaminian** [comment]
> @hagoptaminian, you can now rebase this and mark it ready for review.

👍

File: src/components/space/ActionsMap.js [hunk 1]
```diff
@@ -111,6 +122,13 @@ function ActionsMap() {
                         20,
                       )}px`,
                     }}
+                    onClick={() => handleClusterZoom(longitude, latitude)}
+                    onKeyPress={(event) => {
+                      if (event.keyCode === 13)
```
Comment (juancarlosfarah) [inline src/components/space/ActionsMap.js:127]: Avoid not wrapping `if` statements in `{}`, even if they are just one line.

Comment (juancarlosfarah) [inline src/components/space/ActionsMap.js:127]: Either factor out `13` to a constant `ENTER_KEY_CODE` or add a comment.

Comment (hagoptaminian) [reply src/components/space/ActionsMap.js:127]: 👍

Comment (hagoptaminian) [reply src/components/space/ActionsMap.js:127]: 👍 👍

**Pull request merged**